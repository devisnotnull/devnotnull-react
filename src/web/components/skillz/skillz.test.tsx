import React from 'react';
import { shallow } from 'enzyme';

import Skillz from './skillz';

describe('<Skillz />', () => {
  it('render Skillz Component', () => {
    const wrapper = shallow(<Skillz abilitiesList={[]}></Skillz>);
    expect(wrapper).toMatchSnapshot();
  });
});
