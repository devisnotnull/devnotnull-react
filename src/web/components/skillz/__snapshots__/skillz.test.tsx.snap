// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Skillz /> render Skillz Component 1`] = `ShallowWrapper {}`;
